app/src/main/res/values/setup.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="ExtraTranslation,TypographyDashes,Typos">
-    <string name="default_queue_id">a0b87175-4ed1-44a7-9e78-e798f45d233f</string>
+    <string name="default_queue_id">1d15802e-7ff3-4451-a1ee-9dda99bb75bb</string>
```

Comment:
To double-check, because the credentials question is a sensitive topic. Do we really need to change this value?

Reply:
My mistake. Fixed: 345ce4c